src/resolve/resolved-conf.c

```diff
@@ -599,3 +599,37 @@ int manager_parse_config_file(Manager *m) {
         return 0;
 
 }
+
+int config_parse_dns_cache_size(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        Manager *m = ASSERT_PTR(userdata);
+        int ret, val;
+
+        assert(filename);
+        assert(rvalue);
+
+        if (isempty(rvalue)) {
+                m->cache_size = CACHE_MAX_DEFAULT;
+                return 0;
+        }
+
+        ret = safe_atoi(rvalue, &val);
+
+        if (ret < 0) {
+                log_syntax(unit, LOG_WARNING, filename, line, 1,
+                        "Failed to parse cache size '%s', ignoring, setting value to %d", rvalue, CACHE_MAX_DEFAULT);
+                return 0;
+        } else
+                m->cache_size = val;
```

Comment:
we like shallow indentation. hence drop the word `else` and indent this one level less.

generally we want that the "clean successful code path" is a the top indentation level, and error handling is indented further.